Source/ACE/Entity/Player.cs

```diff
@@ -2345,27 +2345,27 @@ public void HandleActionUse(ObjectGuid usedItemId)
         {
             new ActionChain(this, () =>
             {
-                if (CurrentLandblock != null)
+                if (GetInventoryItem(usedItemId) != null)
                 {
-                    // Just forward our action to the appropriate user...
-                    ActionChain onUseChain = new ActionChain();
-                    CurrentLandblock.ChainOnObject(onUseChain, usedItemId, (WorldObject wo) =>
+                    WorldObject iwo = GetInventoryItem(usedItemId);
+                    ////iwo = iwo.GetObjectFromWeenieType(); // is this right? is this even needed?
+                    iwo.InternalOnUse(Session);
```

Comment:
How about commenting this line, then.  It has a subtle behavior which is not immediately documented.  If I know how the action chains work well, and couldn't immediately see its purpose, it could probably use a comment.